packages/network/src/connection/webRtcConnectionFactories.ts

```diff
@@ -0,0 +1,17 @@
+import { ConstructorOptions, WebRtcConnection } from './WebRtcConnection'
+import { NodeWebRtcConnection } from './NodeWebRtcConnection'
+import nodeDataChannel from 'node-datachannel'
+
+export interface WebRtcConnectionFactory {
```

Comment:
I would suggest removing this file, and moving the WebRtConnectionFactory interface to the WebRtcEndpoint.ts file, and the NodeWebRtcConnectionFactory object to the NodeWebRtcConnection.ts file. This file is uneccessary, and developers of new WebRtcConnection subclasses might feel unconfortable about editing it (as opposod to merely adding new files to the repository)